k6/README.md

```
@@ -0,0 +1,273 @@
+시드 배열 순회:
+  1. docker exec → clean-data.sql (DB 초기화)
+  2. docker exec → CALL seed_all('{seed}') (시드 생성)
+  3. 워밍업 대기 10초
+  4. VU 배열 순회:
+     - k6 run --summary-export → summary JSON 저장
+     - 테스트 간 대기 10초
+```
+
+### parse-results.ps1 내부 흐름
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)